features/admin.server-configurations.v1/forms/username-recovery-form.tsx

```diff
@@ -100,34 +107,99 @@ export const UsernameRecoveryConfigurationForm: FunctionComponent<UsernameRecove
      * @param values - Form values.
      * @returns Sanitized form values.
      */
-    const getUpdatedConfigurations = (value: boolean) => {
-        const data: { [ key: string ]: unknown } = {
-            [ GovernanceConnectorUtils.decodeConnectorPropertyName(
-                ServerConfigurationsConstants.USERNAME_RECOVERY_ENABLE) ]: value.toString()
+    const getUpdatedConfigurations = (values: Record<string, any>) => {
+        const data: UsernamePasswordRecoveryFormUpdatableConfigsInterface = {
+            "Recovery.Notification.Username.Email.Enable": values.enableEmailBasedRecovery !== undefined
+                ? values.enableEmailBasedRecovery
+                : initialConnectorValues?.enableEmailBasedRecovery,
+            "Recovery.Notification.Username.Enable":
+                (values.enableEmailBasedRecovery || values.enableSMSBasedRecovery) !== undefined
+                    ? values.enableEmailBasedRecovery || values.enableSMSBasedRecovery
+                    : (initialConnectorValues?.enableEmailBasedRecovery
+                        || initialConnectorValues?.enableSMSBasedRecovery),
+            "Recovery.Notification.Username.SMS.Enable": values.enableSMSBasedRecovery !== undefined
+                ? values.enableSMSBasedRecovery
+                : initialConnectorValues?.enableSMSBasedRecovery
         };
 
         return data;
     };
 
+    if (!initialConnectorValues){
+        return null;
+    }
```

Comment:
Formatting improvement
```suggestion
    if (!initialConnectorValues) {
        return null;
    }
```

Reply:
Fixed by https://github.com/wso2/identity-apps/pull/7004/commits/ee46b9ff5add8d8e7ec195490774ae655ba9ac4a